one_fm/public/js/doctype_js/job_offer.js

```diff
@@ -128,13 +128,13 @@ frappe.ui.form.on('Job Offer', {
   employee_grade: function(frm) {
     set_filters(frm);
   },
-  offer_term_templates: function(frm){
-    if (frm.doc.offer_term_templates){
+  job_offer_term_template: function(frm){
```

Comment:
this is already handled in core job offer js, so we can remove this

Ref:https://github.com/frappe/hrms/blob/bb22a85001ae8fde6b48fa57cef0a63676abc208/hrms/hr/doctype/job_offer/job_offer.js#L24